source/api/ut_runner.pkb

```diff
@@ -43,6 +43,19 @@ create or replace package body ut_runner is
     return ut_utils.gc_version;
   end;
 
+  function version_compatibility_check( a_requested varchar2, a_current varchar2 := null ) return boolean is
```

Comment:
For boolean is not a SQL type but only PL/SQL (I guess it's still working that way) it might be more complicated to implement the Java request to this function, cause we can't just `select version_compatibility_check(?) from dual`
What do you think about return 0/1? Or adding another function, wrapping the boolean one and returning 0/1?

Reply:
Interesting observation.
We already have bool ->init and int->bool so the change will be quite simple.
I'm just wondering how would the select statement behave with overloaded functions (with boolan and integer return type) - will need to check

Reply:
You can't overload a funtion with a version which differes ony in return type.